docs/src/get-started/installation.rst

```diff
@@ -175,5 +175,5 @@ Other useful configuration options are:
 |                                        | install metatensor-torch?                     |                |
 +----------------------------------------+-----------------------------------------------+----------------+
 
-.. _C++ interface of metatensor: https://lab-cosmo.github.io/metatensor/latest/get-started/installation.html#installing-the-c-and-c-library
-.. _TorchScript interface of metatensor: https://lab-cosmo.github.io/metatensor/latest/get-started/installation.html#for-usage-from-c
+.. _C++ interface of metatensor: https://lab-cosmo.github.io/metatensor/get-started/installation.html#installing-the-c-and-c-library
+.. _TorchScript interface of metatensor: https://lab-cosmo.github.io/metatensor/get-started/installation.html#for-usage-from-c
```

Comment:
I think these still need to contain the `latest`. The redirect only happen on direct access to the `metatensor/` url.

Reply:
Yes, right.